src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,25 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        /// <summary>
+        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
+        /// </summary>
+        /// <returns>A string representation of the current exception.</returns>
+        public override string ToString()
+        {
+            StringBuilder text = new StringBuilder();
+            text.Append(SR.ReflectionTypeLoad_LoadFailed.ToString());
+
+            for (int i = 0; i < LoaderExceptions.Length; i++)
+            {
+                text.Append(Environment.NewLine);
```

Comment:
This is putting 2 new lines between each one.

I would either put 1 or 0 new lines.